build_options.cmake

```diff
@@ -14,7 +14,9 @@ if ( WIN32 ) # TODO: add detection of Windows Phone / Windows Universal platform
 elseif( APPLE AND NOT iOS )
     include( "${CMAKE_CURRENT_LIST_DIR}/toolchains/osx.toolchain.cmake" )
 elseif( ${CMAKE_SYSTEM_NAME} MATCHES "Linux" )
-    include ( "${CMAKE_CURRENT_LIST_DIR}/toolchains/linux.toolchain.cmake" )
+    include( "${CMAKE_CURRENT_LIST_DIR}/toolchains/linux.toolchain.cmake" )
+elseif( ANDROID_TOOLCHAIN ) # TNUN android.toolchain.cmake does not define this variable, while native Android Studio toolchain does
```

Comment:
Essentially yes. Android Studio first forces its own android toolchain which does not have as optimal flags as TNUN android cmake. Since you cannot force AS to use non-default toolchain, this includes a special "android-studio toolchain" which improves flags imposed by AS default toolchain.